variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "project" {
+  default = "Unknown"
+}
+
+variable "environment" {
+  default = "Unknown"
+}
+
+variable "vpc_id" {}
+
+variable "cache_identifier" {}
+
+variable "parameter_group" {
+  default = "memcached1.4"
+}
+
+variable "subnet_group" {}
+
+variable "maintenance_window" {}
+
+variable "desired_clusters" {
+  default = "1"
+}
+
+variable "instance_type" {
+  default = "cache.t2.micro"
+}
+
+variable "engine_version" {
+  default = "1.4.33"
+}
+
+variable "notification_topic_arn" {}
+
+variable "alarm_cpu_threshold" {
+  default = "75"
+}
+
+variable "alarm_memory_threshold" {
```

Comment:
Minor nitpick, on these types of variables I've found it easier to parse the values if the name reflects the unit. In this case maybe `alarm_memory_threshold_bytes`. It avoids having to read the comment or add comments for the variable to make sense in tfvar files then.